packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -177,10 +177,10 @@ class AutoCompleteState extends State<AutoComplete> with AutoDisposeMixin {
                 ))
             .toList();
 
-    const minTileHeight = 48.0;
+    const padding = 8.0;
```

Comment:
use our constands for padding rather than hardcoding 8.0. If we ever tweak our padding that makes it easier.